src/w3c/conformance.js

```diff
@@ -1,13 +1,56 @@
 // @ts-check
 // Module w3c/conformance
 // Handle the conformance section properly.
+import { getIntlData, htmlJoinAnd } from "../core/utils.js";
 import { hyperHTML as html } from "../core/import-maps.js";
-import { htmlJoinAnd } from "../core/utils.js";
 import { pub } from "../core/pubsubhub.js";
 import { renderInlineCitation } from "../core/render-biblio.js";
 import { rfc2119Usage } from "../core/inlines.js";
 export const name = "w3c/conformance";
 
+const localizationStrings = {
+  en: {
+    conformance: "Conformance",
+    normativity:
+      "As well as sections marked as non-normative, all authoring guidelines, " +
+      "diagrams, examples, and notes in this specification are non-normative. " +
+      "Everything else in this specification is normative.",
+    keywordInterpretation(keywords, plural) {
+      return html`
+        <p>
+          The key word${plural ? "s" : ""} ${keywords} in this document
+          ${plural ? "are" : "is"} to be interpreted as described in
+          <a href="https://tools.ietf.org/html/bcp14">BCP 14</a>
+          ${renderInlineCitation("RFC2119")} ${renderInlineCitation("RFC8174")}
+          when, and only when, they appear in all capitals, as shown here.
+        </p>
+      `;
+    },
+  },
+  de: {
+    conformance: "Anforderungen",
+    normativity:
+      "Neben den explizit als nicht-normativ gekennzeichneten Abschnitten " +
+      "sind auch alle Diagramme, Beispiele und Hinweise in diesem Dokument " +
+      "nicht normativ. Alle andere Angaben sind normativ.",
+    keywordInterpretation(keywords, plural) {
+      return html`
+        <p>
+          ${plural ? "Die Schlüsselwörter" : "Das Schlüsselwort"} ${keywords} in
+          diesem Dokument ${plural ? "sind" : "ist"} gemäß
+          <a href="https://tools.ietf.org/html/bcp14">BCP 14</a>
+          ${renderInlineCitation("RFC2119")} ${renderInlineCitation("RFC8174")}
+          und unter Berücksichtigung von
+          <a href="https://goo.gl/6QZH4J">2119de</a>
```

Comment:
```suggestion
          <a href="https://github.com/adfinis-sygroup/2119/blob/master/2119de.rst">2119de</a>
```

No need to use shortened URL here.

BTW, it seems it's not an official W3C translation, am I right? I think we need an official translation, but I would wait for @marcoscaceres to make sure.

Reply:
Is there an official translation? I'd rather use that, but I didn't find any.

Reply:
Yeah, it's a bit unfortunate that there is no official translation. However, we would need to bug the IETF about that. I'm ok with going with this for now tho.